docs/ref/contrib/admin/index.txt

```diff
@@ -1575,7 +1575,7 @@ templates used by the :class:`ModelAdmin` views:
             def get_urls(self):
                 urls = super().get_urls()
                 my_urls = [
-                    path('my_view/', self.my_view),
+                    path('my_view/', self.admin_site.admin_view(self.my_view))
```

Comment:
One possible improvement could be to add a comment above this line (similar to the explanatory comments below) that indicates the necessity of `admin_view()` with a "see note below" type thing.